lib/elements/login_container.dart

```diff
@@ -0,0 +1,145 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:lab1/pages/home_page.dart';
+import 'package:lab1/pages/register_page.dart';
+
+class LoginContainer extends StatefulWidget {
+  const LoginContainer({super.key});
+
+  @override
+  State<StatefulWidget> createState() => _LoginContainerState();
+}
+
+class _LoginContainerState extends State<LoginContainer> {
+  @override
+  Widget build(BuildContext context) {
+    final mediaQuery = MediaQuery.of(context);
+    final minSize = min(mediaQuery.size.width, mediaQuery.size.height);
+    final maxSize = max(mediaQuery.size.width, mediaQuery.size.height);
+
+    final List<Widget> children = [];
+    children.add(
+      Icon(
+        Icons.person,
+        size: minSize * 0.15,
+      ),
+    );
+    children.add(
+      TextField(
+        decoration: InputDecoration(
+          contentPadding: EdgeInsets.symmetric(
+            horizontal: mediaQuery.size.width * 0.03,
+          ),
+          // background: Colors.white,
+          border: OutlineInputBorder(
+            borderRadius: BorderRadius.circular(30),
```

Comment:
better to use `BorderRadius.all(Radius.circular(30)),` it will provide you a `const` option to optimise

Reply:
Fixed, thanks